e2e-tests/pages/insurance/eligibility/checkYourAnswers.js

```diff
@@ -0,0 +1,6 @@
+const checkYourAnswersPage = {
+  warning: () => cy.get('[data-cy="warning"]'),
```

Comment:
Should this be in `pages/shared`? Thinking if we need to use a warning somewhere else.